network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/primary_heroguts.html

```diff
@@ -1,15 +1,23 @@
 {% load wagtailcore_tags wagtailimages_tags static %}
 
 <div id="hero">
-  <div class="banner{% if homepage %} homepage{% endif %}">
-    {% if homepage %}
-      {% comment %}
-        ...homepage-only templating code...
-      {% endcomment %}
-    {% endif %}
-
+  <div class="banner">
     {% with banner=page.specific.get_banner %}
       {% if banner %}
+        {% if homepage %}
+        <picture>
+          {% image banner fill-4032x480 as image_xl %}
+          {% image banner fill-2400x480 as image_lg %}
+          {% image banner fill-1984x480 as image_md %}
+          {% image banner fill-1536x390 as image_sm %}
+          <source media="(min-width: 1200px)" srcset="{{ image_xl.url }}" alt="">
+          <source media="(min-width: 992px)" srcset="{{ image_lg.url }}" alt="">
+          <source media="(min-width: 768px)" srcset="{{ image_md.url }}" alt="">
+          <source media="(min-width: 576px)" srcset="{{ image_sm.url }}" alt="">
+          {# Fallback Image #}
+          <img src="{{ image_sm.url }}" alt="">
```

Comment:
Nice use of `<picture>`! 

Any chance we could make use of the `alt` attrs? It'd be based on each image so I'm not sure that's possible on this site, but if we have the image description somewhere we could use that. If not, please ignore.

Reply:
This one is tricky and I'm leaning towards having a blank `alt=""`. Thought about using image object's `title` for `alt` but since `title` defaults to the image file name and rarely any CMS users updated it to a meaningful title (I checked uploaded images on production), I think using `filename.jpg` as `alt` would be super confusing. Using a generic alt text like "hero banner image" doesn't add much value so that's why I left it blank here. What do you think?

Reply:
That's fair. For some reason I thought we had an `alt` field on images, but we don't. Let's leave these as blank for now, and I'll resolve this comment.